pkg/cmd/cli/cmd/deploy.go

```diff
@@ -228,6 +229,7 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err == nil {
 		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
```

Comment:
This is mainly for discovering other commands and when would it make sense
to use them - oc logs dc in this case

On Fri, Apr 8, 2016 at 5:05 AM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/deploy.go
> <https://github.com/openshift/origin/pull/8360#discussion_r58975438>:
>
> > @@ -228,6 +229,7 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
> >  	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
> >  	if err == nil {
> >  		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
> > +		fmt.Fprintf(out, "Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
>
> Does this really need to be on a new line? Does it look better or worse?
> If you put it on one line, can you shorten the second sentence?
>
> If it's so common that someone wants to follow their deployment with the
> logs, why wouldn't we include that in deploy or do it by default? What
> percentage of users will want to view the logs of the deployment?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8360/files/d2a1659ffa3d63986f3dae6000c482e2c5bc1a6b#r58975438>
>